### PR TITLE
Parse representation file

### PR DESCRIPTION
**Pull request by ErikSchierboom**
See https://github.com/exercism/development-environment/issues/34

### CODE REVIEW

File: app/commands/tooling_job/process.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def call
           attrs["iteration_uuid"],
           attrs["execution_status"],
           "Nothing to report", # TODO
-          download_file(attrs["output"]["representation.txt"]),
+          JSON.parse(download_file(attrs["output"]["representation.txt"])),
```
Comment (ErikSchierboom) [inline app/commands/tooling_job/process.rb:32]: Without this, the data in the DB is stored as a string with double quotes around it.

Comment (iHiD) [reply app/commands/tooling_job/process.rb:32]: Hmmm, does that not seem wrong to you? I suspect this means that the representation.txt is getting uploaded with the extra quotes incorrectly in the first place?

Comment (ErikSchierboom) [reply app/commands/tooling_job/process.rb:32]: It does. It is on my "todo" list, but I wanted to get the PR's up for Mark (mjs) to be able to continue. I'll dig a bit deeper into this.

**Comment by ErikSchierboom** [comment]
This can be closed

**Pull request closed without merging**